cypress/integration/crud.spec.ts

```diff
@@ -0,0 +1,50 @@
+const book_title = 'The Little Schemer';
+
+describe('CRUD Actions', () => {
+	context('Valid Data', function(){
+		beforeEach(() => {
+			cy.visit('/#/examples/crud');
+			cy.get('.add').click();
+			cy.get('#mat-input-0').type(book_title);
+			cy.get('#mat-input-1').type('Friedman{enter}');
+		});
+	  
+		it('should display a book added', () => {
+			cy.get('mat-card>h3').should('contain', book_title);
```

Comment:
Same remark here, to fix this we can add a `data-testid` attribute on the title.